orchestrator/do-release/Jenkinsfile

```diff
@@ -54,7 +54,7 @@ def doRelease(exoUser, jenkinsAgentRootPath, taskID, project, releaseCMD, isInPa
                     --env-file /${jenkinsAgentRootPath}/.eXo/Release/exo-release.properties \
                     --env-file ${config.ENV_FILE_FROM_JENKINS} \
                     -e CATALOG_BASE_URL=${CATALOG_BASE_URL} \
-                    -e MAVEN_OPTS=${MAVEN_OPTS} \
+                    -e MAVEN_EXTRA_OPTS=${MAVEN_EXTRA_OPTS} \
```

Comment:
shouldn't be MAVEN_OPTS as it has included MAVEN_EXTRA_OPTS?

Reply:
nope, we should use MAVEN_EXTRA_OPTS to be concatenated with MAVEN_OPTS at the container entry point otherwise we will overwrite MAVEN_OPTS